main/ptrarray.h

```diff
@@ -0,0 +1,57 @@
+/*
+*   Copyright (c) 1999-2002, Darren Hiebert
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your option) any later version.
+*
+*   Defines external interface to resizable pointer arrays.
+*/
+#ifndef CTAGS_MAIN_PTRARRAY_H
+#define CTAGS_MAIN_PTRARRAY_H
+
+/*
+*   INCLUDE FILES
+*/
+#include "general.h"  /* must always come first */
+
+/*
+*   MACROS
+*/
+#define ptrArrayCount(current) ((current)->count)
+#define ptrArrayItem(current, indx) ((current)->array[indx])
+#define ptrArrayLast(current) ((current)->array[(current)->count - 1])
```

Comment:
I maybe overdid it with the macros - they probably aren't necessary for the pointer array (they make sense for vString because one can really see performance difference between the macro implementation of vStringPut and the function one).